src/main.js

```diff
@@ -63,16 +63,24 @@ function message(msg) {
   }, 3000);
 }
 
+const prefetched = [];
 function prefetch(e) {
   if (e.target.tagName != "A") {
     return;
   }
   if (e.target.origin != location.origin) {
     return;
   }
+  if (window.location.pathname === e.target.pathname) {
```

Comment:
This would also need to compare the query string. So, all of the URL except for the fragment.